dotcom-rendering/src/web/components/AdSlot.tsx

```diff
@@ -541,7 +508,7 @@ export const AdSlot: React.FC<Props> = ({ position, display }) => {
 					data-label="false"
 					data-refresh="false"
 					data-out-of-page="true"
-					data-desktop={[`${Size.outOfPage}`].join('|')}
+					data-desktop={[`${adSizes.outOfPage.toString()}`].join('|')}
```

Comment:
```suggestion
					data-desktop={[adSizes.outOfPage].map(size => size.toString()).join('|')}
```

Reply:
Obvs., this makes more sense in other contexts where there are more items but mayyybe this is a nicer way to do this? Just a thought, I am very happy to go with whatever you prefer

Reply:
This is miles better @oliverlloyd. I'll make the change, thanks!